### ISSUE TITLE
Update to version 4.3.24?

### ISSUE THREAD

**Issue by macuello** [vboxme/Portable-VirtualBox#19]
Can someone tell me what are the changes neeeded in source code to use last version of Virtualbox?

TIA

**Comment by d4k0** [comment]
It seems that the folder structure changed, so the code has to be adjusted:

https://github.com/vboxme/Portable-VirtualBox/issues/18

**Comment by d4k0** [comment]
Good news: I made the changes @LazyZhu proposed (i. e. changing all paths to the new ones) and it seems to work. Here is the changed version of the file "Portable-VirtualBox.au3":

https://app.box.com/s/6ogfk91ibqog0m0f5crwxwyrwwgtjd0a

If you want, I can also upload the updated installer.

**Comment by macuello** [comment]
Yes, please. Can you upload the installer?

Thx again.

**Comment by d4k0** [comment]
Here you go:

https://app.box.com/s/g650t6kx0jmhbc301s7us4ttbyfupr9d

I only tested it with the x64 version of VirtualBox, but there shouldn't be any problems with the x86 version. The installer will only work with VirtualBox 4.3.22 and higher.

**Comment by cocoon** [comment]
v4.3.26 seems to haved changed everything again? ^^ 
multiarch, only 64 bit files and a folder with x86 dlls.

**Comment by d4k0** [comment]
Do you mean the x86 folder in the app64 folder? This seems to be "normal" because the folder is also there when using the old versions of Portable-VirtualBox (the same thing happens when you manually extract the files from the VirtualBox installer). When you look at the code of Portable-VirtualBox, you will notice that it explicitly copies this folder (even if you select the x64 version), so this should be a normal behavior. The question should rather be: Why are the VirtualBox developers doing this? :D

**Comment by cocoon** [comment]
Ah sorry you're right, and yes I hope they will stop chaning it that often, manually it's a pain to extract all the time!

**Comment by macuello** [comment]
d4k0, does your installer work fine with version 4.3.26?

**Comment by d4k0** [comment]
4.3.26 should work just fine, I'm using it myself :).

**Comment by Megachip** [comment]
Could you provide a patch for the master repository?

**Comment by d4k0** [comment]
Yes, I could do this. However, the installer would then only work with version 4.3.22 and higher. Is this OK with you?

**Comment by runarbu** [comment]
That is ok @d4k0 . If needed so can we make a new branch when merging in your patch.

**Comment by d4k0** [comment]
OK, I just created a pull request with the changes.

**Comment by runarbu** [comment]
Thank you @d4k0 . Your patch looks good. Because of Easter I am away, living in a cabin in the woods without any internet, except modem speed on my phone, so it is not possibly for me to release this before I get back. I will however test and release as soon as I get back to civilization.

**Comment by rcdosado** [comment]
hi this is a little off topic but im modifying portable-virtualbox so it can run in a flashdrive, auto starts and automatically load the OS image and ive figured this out and its possible in the settings, i have some problem with the speed of loading of the OS images? and i want to modify it so it will copy the images in a temp folder of the host operating system and load it from there, as it takes so long if the OS image are from the flash drive, i am still learning autoit scripting as of now, and cant find where in the source code ill start reading. can you point me in that direction, anyone?or better if there is someone already implemented this can you give me a link? thank you!

**Comment by runarbu** [comment]
Support for VirtualBox v4.3.26 has been online at vbox.me for some time now. No issues has been reported so we can assume that it work.

**Comment by p060477** [comment]
hi!


i'm running  vbox latest portable 4.3.28 and if i choose network-bridge i Always get at the start :

"network config sample has stop workin" 

then vbox starts and works with my network -bridge opt- 

but if i esc vbox and then i re enter i got the same pop up a/m

BUT then vbox do not find my network so is not able to start..

i esc again and re entering i got Always the same pop up -network config sample has stop work"

but the vbox works well..and so on...


to sum up :


the first time i start vbox i get the pop up error but vbox works

if i esc then

the second time same error but vbox do not work

i esc again

third time....vbox works

fourth  NO

five  YES

and so on....


what have i to do to solve..??


this happens only with win 8.1 as host

with xp as host no


i ask on MS forum but the admin reply that

is a matter of the dev of the sw....


could you pls help me!!


thxs so much in adv!


cheers!!


:)
